checker/tests/nullness/java-unsound/Figure1.java

```diff
@@ -1,5 +1,6 @@
 // Unsound only in Java 8, Java 9+ already gives an error
 // @below-java9-jdk-skip-test
+// @skip-test no need to test for the javac error.
```

Comment:
Should the file be removed (and likewise for the other newly skipped files)?

Reply:
See reply: https://github.com/typetools/checker-framework/pull/4352/files#r587872848